orange-datamodel/src/main/java/com/hartwig/hmftools/datamodel/isofox/GeneExpression.java

```diff
@@ -8,18 +8,20 @@
 @Gson.TypeAdapters
 @Value.Immutable
 @Value.Style(passAnnotations = { NotNull.class, Nullable.class })
-public interface GeneExpression {
-
+public interface GeneExpression
+{
     @NotNull
-    String geneName();
+    String gene();
 
     double tpm();
 
     double medianTpmCancer();
 
     double percentileCancer();
 
-    double medianTpmCohort();
+    @Nullable
+    Double medianTpmCohort();
 
-    double percentileCohort();
+    @Nullable
+    Double percentileCohort();
 }
```

Comment:
Apologies, just realized I mixed up these 2 fields. The `Cancer` fields need to be nullable rather the cohort fields. The cohort fields are "with respect to entire database" and hence are never null. The cancer fields are "with respect to cancer type" which can be null in case specific sample is missing cancer type (or unknown)